solana installer times out while installing

#### Problem
Build in CI times out while downloading the installer binaries

Installer seems to be ~250 MB and is given at most 30 seconds according to the reqwest client default.

This leads to this error while building in CI
```
InstalledDir: /usr/bin
downloading v1.9.2 installer
Error: error decoding response body: operation timed out
Error: Process completed with exit code 1.
```

#### Proposed Solution

Override the timeout to something more reasonable that the 30 seconds default, possibly a few minutes at least?
https://github.com/solana-labs/solana/blob/master/install/src/command.rs#L94-L107

Does that sound correct or is this symptom hiding something else?

--- a/install/src/command.rs
+++ b/install/src/command.rs
@@ -92,7 +92,8 @@ fn download_to_temp(
     let temp_file = temp_dir.path().join("download");
 
     let client = reqwest::blocking::Client::builder()
-        .timeout(Duration::from_secs(360))
+        .connect_timeout(Duration::from_secs(5))
+        .timeout(None)
         .build()?;
 
     let progress_bar = new_spinner_progress_bar();

--- a/install/src/command.rs
+++ b/install/src/command.rs
@@ -92,7 +92,7 @@ fn download_to_temp(
     let temp_file = temp_dir.path().join("download");
 
     let client = reqwest::blocking::Client::builder()
-        .connect_timeout(Duration::from_secs(5))
+        .connect_timeout(Duration::from_secs(30))
         .timeout(None)
         .build()?;
 

--- a/install/src/command.rs
+++ b/install/src/command.rs
@@ -91,7 +91,9 @@ fn download_to_temp(
     let temp_dir = TempDir::new()?;
     let temp_file = temp_dir.path().join("download");
 
-    let client = reqwest::blocking::Client::new();
+    let client = reqwest::blocking::Client::builder()
+        .timeout(Duration::from_secs(360))
+        .build()?;
 
     let progress_bar = new_spinner_progress_bar();
     progress_bar.set_message(format!("{}Downloading...", TRUCK));

